modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/GridCacheQueryFutureAdapter.java

```diff
@@ -60,37 +59,65 @@
     /** Logger. */
     protected static IgniteLogger log;
 
-    /** */
+    /**
```

Comment:
We permit _single-line_ javadoc comments for fields. Please use them (in all changed places in this PR).